data/mods/Magiclysm/Spells/attunements/Permafrost_Mage.json

```diff
@@ -38,5 +38,42 @@
     "material": [ "magical_material" ],
     "flags": [ "AURA", "ONLY_ONE", "TRADER_AVOID", "NO_TAKEOFF", "NONCONDUCTIVE", "OVERSIZE" ],
     "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "emitter": "emit_cold_energy_strong" } ] }
+  },
+  {
+    "id": "permafrost_mage_ice_age",
+    "type": "SPELL",
+    "name": "Ice Age",
+    "description": "Cool down every living creature in range to level his flesh will crack, and their bones will shatter.",
```

Comment:
I know this, but i want to avoid repeating the same word in the same sentence: "their flesh will crack, and their bones will shatter" sound not really good imo.

Reply:
"Cool down every living creature in range to level" Sounds like this is cut of mid sentence to me. Maybe "to **a** level **where** flesh will crack and bones will shatter." would work a bit better?

Reply:
Now its `Cool down every creature in range till their flesh will crack, bones will shatter, and armor will fracture`